subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_security.c

```diff
@@ -471,6 +477,12 @@ int lwm2m_init_security(struct lwm2m_ctx *ctx, char *endpoint)
 #endif /* CONFIG_LWM2M_DTLS_SUPPORT */
 
 #if defined(CONFIG_LWM2M_RD_CLIENT_SUPPORT_BOOTSTRAP)
+	/* If bootstrap is enabled, we should delete the default server instance,
+	 * because it is the bootstrap server that creates it for us.
+	 * Then on a next boot, we load it from flash.
+	 */
+	lwm2m_engine_delete_obj_inst("1/0");
```

Comment:
This feels more like a workaround if we fix an issue of the upstream Zephyr in our utility library.

Reply:
True. It is a workaround, but this is because we don't have enough time before the release to get changes into upstream.